app/views/admin/users/edit.html.haml

```diff
@@ -24,84 +22,129 @@
             Email: Unstolen
           %td
             = "&#x2713;".html_safe if @user.notification_unstolen
-= form_for [:admin, @user] do |f|
-  .row-fluid
-    - if @user.errors.any?
-      #alert-block
-        .alert.alert-error
-          %h4
-            Please fix the following
-            = pluralize(@user.errors.count, "error")
-          %ul
-            - @user.errors.full_messages.each do |msg|
-              %li
-                = msg
-    .span6.padded
-      .form.form-horizontal
-        %h3
-          User attributes
-        .control-group.padded
-          %lable.control-label
+        %tr
+          %td
             Created
-          .controls.convertTime
+          %td.convertTime
             = l @user.created_at, format: :convert_time
-        .control-group.padded
-          %lable.control-label
+        %tr
+          %td
             Updated
-          .controls.convertTime
+          %td.convertTime
             = l @user.updated_at, format: :convert_time
+  - if @user.ambassador?
+    .col-md-6
+      - ambassador = Ambassador.find(@user.id)
+
+      %h1= "Ambassador: #{ambassador.username}"
 
-        .control-group.padded
-          = f.label :name, class: "control-label"
-          .controls
-            = f.text_field :name
-        .control-group
-          = f.label :email, class: "control-label"
-          .controls
-            = f.email_field :email
-        .control-group
-          %label.control-label
-            Secondary Emails
-          .controls
-            = text_area_tag :secondary_emails, @user.secondary_emails.join(", "), disabled: true, rows: 2, style: "width: 100%; font-size: 80%;"
-        .control-group
-          = f.label :username, class: "control-label"
-          .controls
-            = f.text_field :username
-    .span6.padded
-      .form.form-horizontal
-        .control-group
-          = f.label :confirmed, class: "control-label"
-          .controls
-            = f.check_box :confirmed, disabled: @user.confirmed
-        .control-group
-          = f.label :superuser, class: "control-label"
-          .controls
-            = f.check_box :superuser
+      = ambassador.avatar
```

Comment:
avatar should only be included if it isn't the default image

Also, this table list can always be displayed. It isn't exclusive to ambassadors.

Reply:
Also, this could be put into the table list